src/db_api_util.ts

```diff
@@ -501,3 +501,52 @@ export async function logVolunteerVoterClaimToDb(
     client.release();
   }
 }
+
+/**
+ * Updates the Twilio status of the given text message in the DB.
+ *
+ * Additionally, returns the slack channel and message timestamp so we can
+ * react to the message to indicate success/failure
+ */
+export async function logTwilioStatusToDb(
+  messageSid: string,
+  status: string
+): Promise<null | { slackChannel: string; slackMessageTs: string }> {
+  logger.info(`ENTERING DBAPIUTIL.logTwilioStatusToDb`);
+
+  const client = await pool.connect();
+
+  try {
+    const result = await client.query(
+      `
+      UPDATE messages
+      SET
+        twilio_callback_status = $1,
```

Comment:
What are your thoughts on modifying the successfully_sent field as well?

Reply:
I think we can leave `successfully_sent` as is -- to indicate that we sent the message off to Twilio. I'm not sure there's much use in updating both successfully_sent *and* the twilio_status when we get a twilio status callback.

Reply:
Ok great point